features/permissions/permissions.step-definitions.js

```diff
@@ -0,0 +1,152 @@
+const assert = require('assert');
+const { Given, When, Then } = require('cucumber');
+const ENV = require('../../environment-variables');
+const request = require('request-promise-native');
+const btoa = require('btoa');
+let accessToken = "a"; // a variable to store the access token
```

Comment:
Is this assigned to "a" to just have it assigned to something? Unless we have some kind of lint rule set up that prevents this, you should be able to just do `let accessToken;`